Update manage-sharing-gcc.md information on shared devices

alexyeara opened the pull request:
Updating information regarding enabling WB for resource accounts and disabling it for anonymous users in GCC.

alexyeara committed:
Update manage-sharing-gcc.md

Updating information regarding enabling WB for resource accounts and disabling it for anonymous users in GCC.

opbld32 commented:
Learn Build status updates of commit _[32f8e0f](https://github.com/alexyeara/microsoft-365-docs/commits/32f8e0f4ed2ea2dc4436327c011a7605b11b7e66)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[microsoft-365/whiteboard/manage-sharing-gcc.md](https://github.com/alexyeara/microsoft-365-docs/blob/patch-2/microsoft-365/whiteboard/manage-sharing-gcc.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/76d60323-5511-5963-adf9-59162c800b4a/202210111410420905-9929/BuildReport?accessString=1698fd0671bd42c35c844b0b1484b4f9354c51c3e1db1cfbcb9f76b8d969f69a).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/76d60323-5511-5963-adf9-59162c800b4a?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

alexyeara committed:
Update manage-sharing-organizations.md

Updating information regarding enabling WB for resource accounts and disabling it for anonymous users

ghost commented:
[![CLA assistant check](https://cla.opensource.microsoft.com/pull/badge/signed)](https://cla.opensource.microsoft.com/MicrosoftDocs/microsoft-365-docs?pullRequest=9929) <br/>All CLA requirements met.

alexyeara committed:
Update manage-sharing-gcc-high.md

Updating information regarding enabling WB for resource accounts and disabling it for anonymous users

opbld31 commented:
Learn Build status updates of commit _[bec11e3](https://github.com/alexyeara/microsoft-365-docs/commits/bec11e3f183a9d63eb8801d7e2f6e21c33b33686)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[microsoft-365/whiteboard/manage-sharing-gcc.md](https://github.com/alexyeara/microsoft-365-docs/blob/patch-2/microsoft-365/whiteboard/manage-sharing-gcc.md) | :white_check_mark:Succeeded | |
[microsoft-365/whiteboard/manage-sharing-organizations.md](https://github.com/alexyeara/microsoft-365-docs/blob/patch-2/microsoft-365/whiteboard/manage-sharing-organizations.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/76d60323-5511-5963-adf9-59162c800b4a/202210111414569451-9929/BuildReport?accessString=3c75100a202e4d67f9d71c131127aafd2bc8a2b1ababac124887e3866ee30b39).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/76d60323-5511-5963-adf9-59162c800b4a?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

opbld30 commented:
Learn Build status updates of commit _[c82a17e](https://github.com/alexyeara/microsoft-365-docs/commits/c82a17edffe7f3ed56872284d91cc55767f336c9)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[microsoft-365/whiteboard/manage-sharing-gcc.md](https://github.com/alexyeara/microsoft-365-docs/blob/patch-2/microsoft-365/whiteboard/manage-sharing-gcc.md) | :white_check_mark:Succeeded | |
[microsoft-365/whiteboard/manage-sharing-gcc-high.md](https://github.com/alexyeara/microsoft-365-docs/blob/patch-2/microsoft-365/whiteboard/manage-sharing-gcc-high.md) | :white_check_mark:Succeeded | |
[microsoft-365/whiteboard/manage-sharing-organizations.md](https://github.com/alexyeara/microsoft-365-docs/blob/patch-2/microsoft-365/whiteboard/manage-sharing-organizations.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/76d60323-5511-5963-adf9-59162c800b4a/202210111416523324-9929/BuildReport?accessString=986b47773b4d18974d2720140c8a2c031b7e27bf265e29bf61a1c5a9aca4e822).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/76d60323-5511-5963-adf9-59162c800b4a?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

chuckedmonson committed:
Merge branch 'public' into patch-2

opbld31 commented:
Learn Build status updates of commit _[1c55322](https://github.com/alexyeara/microsoft-365-docs/commits/1c553227a68962e2a5c82c488529b4d3aca502cf)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[microsoft-365/whiteboard/manage-sharing-gcc.md](https://github.com/alexyeara/microsoft-365-docs/blob/patch-2/microsoft-365/whiteboard/manage-sharing-gcc.md) | :white_check_mark:Succeeded | |
[microsoft-365/whiteboard/manage-sharing-gcc-high.md](https://github.com/alexyeara/microsoft-365-docs/blob/patch-2/microsoft-365/whiteboard/manage-sharing-gcc-high.md) | :white_check_mark:Succeeded | |
[microsoft-365/whiteboard/manage-sharing-organizations.md](https://github.com/alexyeara/microsoft-365-docs/blob/patch-2/microsoft-365/whiteboard/manage-sharing-organizations.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/76d60323-5511-5963-adf9-59162c800b4a/202210111652318130-9929/BuildReport?accessString=2dded8614744d566fee2af608ed1c1a942499fda71c82f0099f4ed40f4ef8700).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/76d60323-5511-5963-adf9-59162c800b4a?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>
